README.md

```diff
@@ -1,299 +1,75 @@
-    -   Should be type-dependent, rather than mandating "materials" and
-        "products."
-
-## Reasoning
-
-### Reason for separate Statement and Predicate layers
-
-The [Statement] layer has a fixed schema while the [Predicate] layer has an
-arbitrary schema. Furthermore, the fixed Statement schema has a `subject` and
-`predicateType`. There are two main reasons for this.
-
-First, doing so allows policy engines to make decisions without requiring
-Predicate-specific logic or configuration. Binary Authorization policies today
-are purely about "does an attestation exist that is signed by X with subject Y",
-and similarly in-toto layouts are about "does an attestation exist that is
-signed by X with materials/products Z?"[1] These relatively simple policies are
-quite powerful. With this proposal, such policies become more expressive without
-any additional configuration: "does an attestation exist that is signed by X
-having predicate type T, with subject Y/Z?"
-
-Second, it enables lookup of attestations by `subject`, again without
-Predicate-specific logic or configuration. Consider the policy described in the
-[motivating use case](#motivating-use-case). There, the instruction is "fetch
-attestations for artifact X". The lookup could be from a set of attestations
-provided by the caller, or it could be from an external database keyed by
-subject.[2] Without a standardized `subject` field, this would be significantly
-harder.
-
-The alternative is to not have a fixed Statement schema and instead have
-`subject` be part of the Predicate. Doing so would require users to configure
-the system for every possible Predicate type they wanted to support, in order to
-instruct the system how to find the subject. Furthermore, because there would be
-no standardization, concepts and models may not necessarily translate between
-predicate types. For example, one predicate type might require an "or" between
-artifact IDs, while another requires an "and." This difference would add
-complexity and confusion.
-
-## Footnotes
-
-\[1]: The `expected_command` is only a warning, and `inspections` require
-running external commands which is infeasible in many situations.
-
-\[2]: That said, we strongly recommend against keying a database purely by
-content hash. The reason is that such databases quickly run into scaling issues,
-as explained in
-[Building Secure and Reliable Systems](https://static.googleusercontent.com/media/landing.google.com/en//sre/static/pdf/Building_Secure_and_Reliable_Systems.pdf#page=364),
-Chapter 14, page 328, "Ensure Unambiguous Provenance." Instead, we recommend
-keying primarily by resource name, in addition to content hash.
+# In-toto Attestation Framework
+
+<!-- msm: This is real clunky -->
+The in-toto Attestation Framework is a framework for validating claims about
+any aspect of how a piece of software is produced, allowing anyone
+to verify the origins of software and establish trust in its supply chain.
```

Comment:
Are we providing a framework for validating claims, or a framework for making claims which can be validated? The SLSA docs on the [attestation model](https://slsa.dev/attestation-model) say that:

> A software attestation is an authenticated statement (metadata) about a software artifact or collection of software artifacts.
>
> The primary intended use case is to feed into automated policy engines, such as [in-toto](https://in-toto.io/) and [Binary Authorization](https://cloud.google.com/binary-authorization).

Reply:
I wanted to up-level the description of the framework a bit more, but good point. Right now our focus is on specifying/generating these verifiable claims. I'll change the wording on this, thanks!